tests/test_models.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from django.contrib.sites.models import Site
+
+from controlled_vocabularies.models import Vocabulary, Term, Property
+
+
+def create_vocab(name='languages', label='Language', order='name',
+                 maintainer='Buddy Noone', maintainerEmail='noone@nowhere.com',
+                 definition='Lorem ipsum dolor sit amet'):
+    """Creates an unsaved instance of the Vocabulary model."""
+    return(Vocabulary(name=name,
+                      label=label,
+                      order=order,
+                      maintainer=maintainer,
+                      maintainerEmail=maintainer,
+                      definition=definition))
+
+
+def create_term(name='spanish', label='Spanish', order=2):
+    """Creates an unsaved instance of the Term model."""
+    vocab_instance = create_vocab()
+    vocab_instance.save()
+    return(Term(vocab_list=vocab_instance,
+                name=name,
+                label=label,
+                order=order))
+
+
+def create_property(property_name='dialect', label='Catalan'):
+    """Creates an unsaved instance of the Property model."""
+    term_instance = create_term()
+    term_instance.save()
+    return(Property(term_key=term_instance,
+                    property_name=property_name,
+                    label=label))
+
+
+@pytest.mark.django_db
+class TestVocabulary:
+    def test_save(self):
+        vocab_instance = create_vocab()
+        assert Vocabulary.objects.all().count() == 0
+
+        vocab_instance.save()
+        assert Vocabulary.objects.all().count() == 1
+
+    def test_unicode(self):
+        vocab_instance = create_vocab()
+        assert unicode(vocab_instance) == 'languages'
+
+    @pytest.mark.xfail(reason='Input is not being cleaned before being saved.')
+    def test_whitespace_stripped(self):
```

Comment:
Since this is a failing test put in here kind of reminding us of something we need to fix, it is probably not a big deal how it is implemented at this point, since it can be adjusted when the issue is fixed based on how that is implemented. As far as that goes, if the white space is stripped for all fields using the same model method, it makes sense to me to have it tested in a single test. So I think it is ok to have multiple asserts here. Though the creation and saving and getting in the current test could be condensed.